components/Layout.tsx

```diff
@@ -8,8 +8,9 @@ export default function Layout({ children, page }) {
   const title = "Code Youngstown";
   const description =
     "Connecting software engineers, developers, and coders in the Youngstown, OH area. Join the ongoing chat via Slack and attend our meetups.";
-  const pageTitle = `${title} | ${page}`;
 
+  let pageName = page == "Home" ? "" : `${page} |`;
+  const pageTitle = `${pageName} ${title}`;
```

Comment:
Sorry to be so pedantic, but won't that render the home page title as ` Code Youngstown`, like with an extra space in front?

Reply:
Ironically, GitHub markdown doesn't even render the extra space that I put in there.

Reply:
In localhost and vercel dev it does not appear to, so I am not entirely sure

Reply:
Let's correct it, just in case. Could just be something like

```
const pageTitle = "Home" ? title : `${pageName} ${title}`;
```

Reply:
had to add checking for page prop:
`const pageTitle = page == "Home" ? title : `${page} | ${title}`;`